packages/sdk/src/entities/defaultTokens.ts

```diff
@@ -59,9 +65,16 @@ export const WETH: Readonly<Record<ChainId, Token>> = {
     "WETH",
     "Wrapped Ether"
   ),
+  [ChainId.BASE]: new Token(
+    ChainId.BASE,
+    "0x4200000000000000000000000000000000000006",
+    18,
+    "WETH",
+    "Wrapped Ether"
+  ),
 };
 
-export const SWPR: Readonly<Record<ChainId, Token>> = {
+export const SWPR: Record<Exclude<ChainId, ChainId.BASE>, Token> = {
```

Comment:
Is it worth to create a new Type (local at least in this file) for this Exclusion?

Reply:
nope, I don't think so